scripts/version_mgr.py

```diff
@@ -5,10 +5,12 @@
 import json
 import argparse
 import urllib.request
+import datetime
 
 from git import Git
 
 VERSIONS_FILE = "versions.json"
+WEEKLY_FILE   = "weeklys.json"
```

Comment:
we can continue using one file and just add a flag per entry signifying a weekly milestone.

Reply:
and the change needs to happen on zephyr tree which does create the file